QRCoderTests/PngByteQRCodeRendererTests.cs

```diff
@@ -113,7 +113,6 @@ public void can_render_pngbyte_qrcode_color_without_quietzones()
         var result = HelperFunctions.ByteArrayToHash(pngCodeGfx);
         result.ShouldBe("07f760b3eb54901840b094d31e299713");
 #else
-        File.WriteAllBytes(@"C:\Temp\pngbyte_35.png", pngCodeGfx);
```

Comment:
seems to be a temporary line of code, crashing local tests when `C:\Temp` does not exist

Reply:
Yep, temporary line of code for manually checking the output.